recipes/assimulo/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.9" %}
+
+package:
+    name: assimulo
+    version: {{ version }}
+
+source:
+  fn: Assimulo-{{ version }}.zip
+  url: https://pypi.python.org/packages/4c/c0/19a54949817204313efff9f83f1e4a247edebed0a1cc5a317a95d3f374ae/Assimulo-{{ version }}.zip
+  md5: 3f28fd98011d2ec7a01703a1ef1dff45
+  patches:
+    # no fortran compiler on windows, unsupported on osx
+    - disable-fortran-extensions.patch  # [win or osx]
+
+build:
+  number: 0
+  features:
+  - vc9  # [win and py27]
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - cython
+    - sundials
+    - gcc  # [unix]
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - matplotlib
+    - sundials
+
+test:
+  imports:
+    - assimulo
+
+  requires:
+      - nose
+about:
+  home: http://www.jmodelica.org/assimulo
+  license: GPLv3, LGPLv3, LGPLv2
+  summary: A package for solving ordinary differential equations and differential algebraic equations.
+
+extra:
+  recipe-maintainers:
+    - jschueller
```

Comment:
Yes, I expect him to react on this too, but he doesn't seem to have a github account though